src/assets/image/icons/connections/index.ts

```diff
@@ -13,6 +13,7 @@ export { default as GoogleGmailIcon } from "@assets/image/icons/connections/Goog
 export { default as GoogleSheetsIcon } from "@assets/image/icons/connections/GoogleSheets.svg?react";
 export { default as GrpcIcon } from "@assets/image/icons/connections/Grpc.svg?react";
 export { default as HttpIcon } from "@assets/image/icons/connections/Http.svg?react";
+export { default as HubspotIcon } from "@assets/image/icons/connections/Hubspot.svg?react";
```

Comment:
@J1za please add a link for the icon source 🙏🏻

Reply:
Its added inside svg file

Reply:
Please add inside the index.ts

Reply:
There are many variations in svgrepo.com, we better save the full link to the icon, in order to be able to see the license from there easily.

Reply:
added

Reply:
Please change the icon to this one:
https://www.svgrepo.com/svg/443148/brand-hubspot

Reply:
changed

Reply:
Thanks!